array-manipulation-2/src/take-right.js

```diff
@@ -0,0 +1,18 @@
+/* exported takeRight */
+function takeRight(array, count) {
+  var output = [];
+  for (var i = 0; i < array.length; i++) {
+    if (i > (array.length - count - 1)) {
+      output.push(array[i]);
+    }
+  }
```

Comment:
You can simplify this code a bit by
-  using a starting index other than zero
or
-  looping backwards.